src/base/bittorrent/torrentimpl.h

```diff
@@ -244,6 +254,9 @@ namespace BitTorrent
 
         void handleFastResumeRejectedAlert(const lt::fastresume_rejected_alert *p);
         void handleFileCompletedAlert(const lt::file_completed_alert *p);
+#if (LIBTORRENT_VERSION_NUM >= 20003)
+        void handleFilePrioAlert(const lt::file_prio_alert *p);
```

Comment:
Seems this is a `const` function.

Reply:
I'm sorry, but I'm against determining the constancy of a method just based on its current implementation.